staticd/static_vrf.c

```diff
@@ -51,6 +55,37 @@ static void zebra_stable_node_cleanup(struct route_table *table,
 			static_path_list_del(&si->path_list, pn);
 			XFREE(MTYPE_STATIC_PATH, pn);
 		}
+
+		/* clean up for dst table */
+		src_table = srcdest_srcnode_table(node);
+		if (src_table) {
+			/* This means the route_node is part of the top
+			 * hierarchy and refers to a destination prefix.
+			 */
+			for (src_node = route_top(src_table); src_node;
+			     src_node = route_next(src_node)) {
+				src_si = src_node->info;
+
+				frr_each_safe (static_path_list,
```

Comment:
frr_each_safe(

Reply:
Done.

Actually I clicked on style suggestions, It gives me a suggestion with space and I selected to go with suggestions.
I will check why polychaeta gives this suggestion.

Thanks for review.